osfoffline/polling_osf_manager/polling.py

```diff
@@ -6,6 +6,7 @@
 
 import aiohttp
 import iso8601
+from sqlalchemy.exc import SQLAlchemyError
```

Comment:
Started to make the `save`s safer, but those exceptions should probably be raised. I'll remove this